ipwb/assets/admin.css

```diff
@@ -11,3 +11,6 @@ details label#daemonStatusLabel {height: 10px; margin-top: 0; vertical-align: to
 details label.twoRowLabel {height: 2.0em;}
 
 form input[type=submit] {display: block; clear: both; margin: 0.5em auto 0 auto;}
+h1 {font-size: 24px; margin-bottom: 0.5em; font-family: sans-serif;}
+h1 img {height: 2.0em;}
+ul#uriList li {display: inline;}
```

Comment:
Do we really want to keep the list inline?

Reply:
@ibnesayeed I anticipate the design to be heavily refactored, e.g., #600. Inline here is to override the value pulled in from webui.css, which dictates that the values are initially hidden. We can revise this but (Step 1) Get the values display per the GitHub issue.

Reply:
Resolved in 34882fe42bcd20d8a7b1292d9395fe6524559abd

Reply:
To undo the effect of an inherited CSS value in this case you could have used `display: list-item;` or `display: initial;`.